test/util/common.hpp

```diff
@@ -0,0 +1,12 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+// Unit Test
+
+// Copyright (c) 2014, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Menelaos Karavelas, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+#define BOOST_GEOMETRY_TEST_USE_UTF
+#include <geometry_test_common.hpp>
```

Comment:
Since this is a header file you need also guard it an appropriate `#ifndef/#define/#endif` block.